src/App.jsx

```diff
@@ -27,7 +28,7 @@ const App = () => (
             />
           </Switch>
         </main>
-        <Footer logo={process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} />
+        <Footer logo={getConfig().LOGO_TRADEMARK_URL || process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} />
```

Comment:
Why you didn't change this line https://github.com/openedx/frontend-component-footer/blob/master/src/components/Footer.jsx#L59   ?

Reply:
Because in that line I have to modify frontend-component-footer. That package is in NPM, and yes is possible to make a fork starting for the actual version but my concern is that to keep it to upstream that component could affect all MFEs. So keeping it here I fixed it in the grade book where I found the footer logo was not runtime. Learning and discussions seem working...